math-scala/src/main/scala/org/apache/mahout/math/cf/SimilarityAnalysis.scala

```diff
@@ -127,12 +129,11 @@ object SimilarityAnalysis extends Serializable {
     retIDSs.toList
   }
 
-  /** Calculates row-wise similarity using the log-likelihood ratio on AA' and returns a drm of rows and similar rows
+  /** Calculates row-wise similarity using the log-likelihood ratio on AA' and returns a DRM of rows and similar rows
     * @param drmARaw Primary interaction matrix
     * @param randomSeed when kept to a constant will make repeatable downsampling
     * @param maxInterestingSimilaritiesPerRow number of similar items to return per item, default: 50
     * @param maxNumInteractions max number of interactions after downsampling, default: 500
-    * @return
```

Comment:
this is Mahout's position on this with javadoc style as well afaik. Not to leave empty @@tags if they are already obvious.